shared/js/participantStore.js

```diff
@@ -119,6 +120,19 @@ loop.store.ParticipantStore = function() {
         }
       }
 
+      if (!updatedParticipant.hasNotifiedOfJoin &&
+          updatedParticipant.isHere &&
+          updatedParticipant.participantName &&
```

Comment:
As in, set up with no name & try to sneak into the room. Unlikely, but possible ;-)